tests/e2e/p/workflow.go

```diff
@@ -94,8 +94,8 @@ var _ = e2e.DescribePChain("[Workflow]", func() {
 
 			// Use a random node ID to ensure that repeated test runs
 			// will succeed against a persistent network.
-			randomShortID, _ := ids.ToShortID(utils.RandomBytes(20))
-			validatorID := ids.NodeID(randomShortID)
+			validatorID, err := ids.ToNodeID(utils.RandomBytes(ids.NodeIDLen))
```

Comment:
I think ideally we should be able to use `ids.GenerateTestNodeID()`. Generation of IDs should result in different results for different runs imo.